app/models/katello/pool.rb

```diff
@@ -50,8 +50,15 @@ def active?
       active
     end
 
+    # used for notification bell
     def expiring_soon?
-      (end_date.to_date - Date.today) <= Setting[:expire_soon_days]
+      expiring_in_days >= 0 &&
+        expiring_in_days <= Setting[:expire_soon_days].to_i
+    end
+
+    # used for entitlements report template
+    def expiring_in_days
```

Comment:
This method name leads me to believe it expects a parameter named 'days' even though that makes no sense because it's an instance method and we're looking at a single pool rather than querying a set.  What do you think about `days_until_expiration`